pipelines/sqre/verify_drp_metrics.groovy

```diff
@@ -243,6 +243,15 @@ def void verifyDataset(Map p) {
         archiveDir: jobDir,
       )
 
+      util.runVerifyToSasquatch(
+        runDir: runDir,
+        gen3Dir: util.joinPath(datasetDir, SMALL_HSC)
+        collectionName: "jenkins/step3"
+        namespace: "lsst.verify.drp",
+        datasetRame: "HSC/RC2/Nightly"
+        sasquatchURL: sqre.sasquatch.url
```

Comment:
These could be placed in https://github.com/lsst-dm/jenkins-dm-jobs/blob/main/etc/scipipe/verify_drp_metrics.yaml rather than hardcoded.

Also, `datasetRame` → `datasetName` is a typo and `sasquatchURL` → `sasquatchUrl`.